Source/Kesmai.Server/Game/Items/Equipment/Weapons/Flails/WoodenFlail.cs

```diff
@@ -17,10 +17,10 @@ public partial class WoodenFlail : Flail
 		public override int MinimumDamage => 1;
 
 		/// <inheritdoc />
-		public override int MaximumDamage => 6;
+		public override int MaximumDamage => 1;
 
 		/// <inheritdoc />
-		public override int BaseArmorBonus => 1;
+		public override int BaseArmorBonus => 10;
```

Comment:
I think you got the wrong property for the 10 on this one

Reply:
Oof yep.

Reply:
corrected.